### PR TITLE
Update vivaldi.rb

### PR DESCRIPTION
**Pull request by gurghet**
Updated to latest version

### CODE REVIEW

File: Casks/vivaldi.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask 'vivaldi' do
-  version '1.0.303.52'
-  sha256 :no_check # required as upstream package is updated in-place
+  version '1.0.344.37'
+  sha256 :no_check # maybe required as upstream package was updated in-place
```
Comment (vitorgalvao) [inline Casks/vivaldi.rb]: Please don’t change the comment. Either add a `sha256` if you’re sure it won’t break, or leave it be.

**Comment by vitorgalvao** [comment]
Please [squash your commits](http://davidwalsh.name/squash-commits-git).

**Comment by gurghet** [comment]
Done.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 30eb6efad275826df087b9e56def1e6706f76cd4 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**